platforms/browser/www/phonegap/plugin/facebookConnectPlugin/facebookConnectPlugin.js

```diff
@@ -161,16 +161,31 @@ cordova.define("com.phonegap.plugins.facebookconnect.FacebookConnectPlugin", fun
         
         // Bake in the JS SDK
         (function () {
+            // Retrieve the root element to append the script tags to. Use the body if no fb-root was found
+            var root = document.getElementById('fb-root') || document.getElementsByTagName('body')[0];
+
             if (!window.FB) {
                 console.log("launching FB SDK");
                 var e = document.createElement('script');
-                e.src = document.location.protocol + '//connect.facebook.net/en_US/sdk.js';
                 e.async = true;
-                document.getElementById('fb-root').appendChild(e);
+                e.src = document.location.protocol + '//connect.facebook.net/en_US/sdk.js';
+                e.onload = e.onreadystatechange = e.onerror = loadLocalSDK;
+
+                root.appendChild(e);
+            }
+
+            /**
+             * If something happens when loading the remote script, then check if
+             * window.FB exists, if not, load the local script.
+             */
+            function loadLocalSDK() {
                 if (!window.FB) {
-                    // Probably not on server, use the sample sdk
+                    var e = document.createElement('script');
+                    e.async = true;
                     e.src = 'phonegap/plugin/facebookConnectPlugin/fbsdk.js';
```

Comment:
I already noticed that the local file does not exist. So it can be added for the use case FB stops serving the file or if FB is down. But then the SDK probably can't make a connection with the FB servers either so... Not sure if a fallback should be implemented.